vcd/resource_vcd_nsxt_ip_set.go

```diff
@@ -33,21 +32,27 @@ func resourceVcdNsxtIpSet() *schema.Resource {
 			"vdc": {
 				Type:        schema.TypeString,
 				Optional:    true,
-				ForceNew:    true,
+				Computed:    true,
```

Comment:
What happens if `vdc` is set at provider level, but here `edge_gateway_id` is set: do we have a test which checks that VDC from EG takes priority?

CC: @Didainius

Reply:
we know VDC from edge gateway. VDC is absolete when edge gateway ID provided.

Reply:
Given that this behavior is fresh in the codeline, what about adding a test for this use case?

Reply:
TestAccVcdNsxtIpSetInheritedVdc covers this case

Reply:
There is an issue with that test. It is using `vdc` field which was deprecated:
https://github.com/vmware/terraform-provider-vcd/pull/809#discussion_r838483026

Reply:
It is not an issue, this is the idea to confirm that everything still works, as we support back Compatability VDC+edgeGtwID and only edgeGtwID ways.

Reply:
Good things are tested with `TestAccVcdNsxtIpSetInheritedVdc`! While for my original request at the top, we would have to add tests with two different VDCs (one at provider level and one in resources), which is very costly from run time perspective.